File: kitchen-web/src/app/model/chapter.ts
import { Book } from './book';

export class Chapter {
    id: number;
    name: string;
    description: string;
    duration: string;
    seqNo: number;
    book: Book;
    createdBy: string;
    creationDate: Date;
    constructor(book: Book) {
        this.id = 0;
        this.name = '';
        this.description = '';
        this.duration = '';
        this.seqNo = 0;
        this.book = book;
        this.createdBy = 'FrontEnd';
        this.creationDate = new Date();
    }
}

File: kitchen-chapter-api/Models/Chapter/Chapter.cs
using System;
using MongoDB.Bson.Serialization.Attributes;
namespace kitchen_api_chapter.Models
{
    public class Chapter
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public string Description { get; set; }
        public string Duration { get; set; }
        public int SeqNo { get; set; }
        public Book Book { get; set; }
        public string CreatedBy { get; set; }
        public DateTime CreationDate { get; set; }
    }
}